src/qt/utilitydialog.h

```diff
@@ -22,7 +22,7 @@ class HelpMessageDialog : public QDialog
     Q_OBJECT
 
 public:
-    explicit HelpMessageDialog(QWidget *parent, bool about);
+    explicit HelpMessageDialog(QWidget* parent, bool about, bool wallet_enabled = false);
```

Comment:
I see default changed to `= false`, but do we really need it (default value) at all?

I mean, why some "abstract" user of this class would allow himself to skip passing explicit argument and let default be assumed as `false`? Maybe there's argument for it and I am missing it.

Reply:
When `about` is `true` it does not matter what value `wallet_enabled` has. It seems weird to choose any its value on the caller site.

Reply:
Oh, so it depends on `about` too. OK got it.